saucal/woocommerce-gateway-amazon-pay
# Feature/checkout unit test

### All Submissions:






### Changes proposed in this Pull Request:

Adds a base for PHP unit tests. Adds a few basic unit tests.



### How to test the changes in this Pull Request:

1. Execute ./tests/bin/install-phpunit-tests-dependencies.sh
2. Execute npm run test:php
3. Tests should be succesfull.

### Other information:




### Changelog entry

> No need for changelog entry, unit tests won't be included in production plugin build.
